src/openrct2/Context.cpp

```diff
@@ -1163,6 +1168,23 @@ namespace OpenRCT2
 
                 Draw();
             }
+
```

Comment:
Could this code here replace the whole RunFixedFrameFunction as we can just pass the MaxFPS value of 40?